core/icons/aliases.json

```diff
@@ -1,10 +1,49 @@
 {
-  "delete": ["trash", "remove"],
+  "database": ["110"],
```

Comment:
Adding budicon numbers to improve searchability in docs 🤩 very clever

Reply:
Don't remove the old ones though 😢

Reply:
Nope! There are some re-arrangements on the file, since it's manual. It's error prone ... but I'm keeping them all (but moving some defs around the file)